### ISSUE TITLE
MAML mini-ImageNet example normalises by number of datapoints twice

### ISSUE THREAD

**Issue by IsakFalk** [learnables/learn2learn#190]
In the example of [maml on mini-imagenet](https://github.com/learnables/learn2learn/blob/master/examples/vision/maml_miniimagenet.py) the `reduction` argument of the cross entropy loss is set to "mean", but in the `fast_adapt` function

```python
def fast_adapt(batch, learner, loss, adaptation_steps, shots, ways, device):
    data, labels = batch
    data, labels = data.to(device), labels.to(device)

    # Separate data into adaptation/evalutation sets
    adaptation_indices = np.zeros(data.size(0), dtype=bool)
    adaptation_indices[np.arange(shots*ways) * 2] = True
    evaluation_indices = torch.from_numpy(~adaptation_indices)
    adaptation_indices = torch.from_numpy(adaptation_indices)
    adaptation_data, adaptation_labels = data[adaptation_indices], labels[adaptation_indices]
    evaluation_data, evaluation_labels = data[evaluation_indices], labels[evaluation_indices]

    # Adapt the model
    for step in range(adaptation_steps):
        adaptation_error = loss(learner(adaptation_data), adaptation_labels)
        adaptation_error /= len(adaptation_data)
        learner.adapt(adaptation_error)

    # Evaluate the adapted model
    predictions = learner(evaluation_data)
    evaluation_error = loss(predictions, evaluation_labels)
    evaluation_error /= len(evaluation_data)
    evaluation_accuracy = accuracy(predictions, evaluation_labels)
    return evaluation_error, evaluation_accuracy
```
the errors are divided by the number of evaluation data points. Doesn't this mean we are dividing by this number twice?

**Comment by seba-1511** [comment]
Hello @IsakFalk,

Correct, that's a mistake. (Note: not too important, since it's a matter of rescaling the fast learning rate.) Do you want to create a PR to fix it?

**Comment by IsakFalk** [comment]
Hi @seba-1511,

Agreed, it's a very small mistake. I mainly pointed it out since it made me was doubt my knowledge of pytorch. I will make a PR sometime tomorrow.

**Comment by steremma** [comment]
Same issue in the OmniGlot example, proposed fix in open PR
